TCL/polarDensity_for_DTA.tcl

```diff
@@ -504,45 +394,93 @@ proc theta_histogram {singleFrame_lower singleFrame_upper  Ntheta } {
     return $theta_bin_out
 }
 
-# TODO I don't think I need this function anymore
-proc theta_clean_up { theta_bin_low theta_bin_high shel_count  Ntheta delta_frame low_f upp_f} {
-    
-    theta_bin_out [list ]
-    
-    foreach ud [list  $theta_bin_high $theta_bin_low] {
-        #Species_Total_Warning $sel_num $shel_count
-        puts "Cleaning up for shell $ri to $rf"
-        #cleanup and output 
-        set theta_bin_counts [lcount $ud]
-        #Shell_Test $shel_count $theta_bin_counts
-        set theta_bin_time_averages {}
-        for {set ti 0} { $ti<=$Ntheta} {incr ti 1} {
-            set tindex [lsearch [lindex $theta_bin_counts 0] $ti]
-            if { $tindex >= 0} {
-                set time_average [expr 1.0 * [lindex [lindex $theta_bin_counts 1] $tindex]/(1.0*($delta_frame))] 
-            } else { 
-                set time_average 0.0
-            }
-            lappend theta_bin_time_averages $time_average
+
+#bins a single shell over many frames
+proc bin_over_frames {shell species headname tailname lipidbeads_selstr dtheta start end Ntheta dt ri rf  flower fupper leaflet_algorithm} {
+    set theta_bin_high [lrepeat [expr $Ntheta+1] 0]
+    set theta_bin_low [lrepeat [expr $Ntheta+1] 0]
+    for {set frm $start} {$frm < ${end}} {incr frm $dt} {
+        $shell frame $frm
+        $shell update 
+        set singleFrame_counts [bin_frame $shell $species $headname $tailname $lipidbeads_selstr $dtheta $frm $leaflet_algorithm ]
+        set singleFrame_upper [lindex $singleFrame_counts 1] 
+        set singleFrame_lower [lindex $singleFrame_counts 0]
+        set theta_bins [theta_histogram $singleFrame_lower $singleFrame_upper  $Ntheta]
+        
+        # should be fixed, do not change [lrepeat [expr $Ntheta+1] to [lrepeat [expr $Ntheta] 
```

Comment:
Can you clarify what's been "fixed" @gbrannigan?

Reply:
Nope! This seems like an echo from the ancient past.

Reply:
But I can remove it